download-basket/src/main/java/downloadbasket/helpers/DownloadServices.java

```diff
@@ -0,0 +1,213 @@
+package downloadbasket.helpers;
+
+import fi.nls.oskari.log.LogFactory;
+
+import fi.nls.oskari.log.Logger;
+import fi.nls.oskari.util.PropertyUtil;
+import fi.nls.oskari.util.IOHelper;
+import downloadbasket.data.ErrorReportDetails;
+import downloadbasket.data.LoadZipDetails;
+import org.apache.commons.mail.MultiPartEmail;
+
+import javax.activation.DataHandler;
+import javax.activation.DataSource;
+import javax.activation.FileDataSource;
+import javax.mail.internet.MimeBodyPart;
+import javax.mail.internet.MimeMultipart;
+import javax.mail.util.ByteArrayDataSource;
+import java.io.*;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.util.UUID;
+import java.util.Locale;
+import java.util.zip.ZipException;
+import java.util.zip.ZipFile;
+import org.springframework.context.MessageSource;
+import fi.nls.oskari.spring.SpringContextHolder;
+
+/**
+ * Download services.
+ *
+ */
+public class DownloadServices {
+	public static final String WFS_GETCAPABILITIES_XML_JSON_VALUE = "WFSGetCapabilitiesXML";
+	public static final String WFS_METADATA_URL_JSON_VALUE = "linkForMetadata";
+	public static final String WFS_USED_URL = "url";
+	public static final String WFS_FEATURETYPES = "featureTypes";
```

Comment:
Looks like none of these constants are referenced in the code. Please remove them.